SwiftAudio/Classes/AudioItem.swift

```diff
@@ -36,6 +36,11 @@ public protocol InitialTiming {
     func getInitialTime() -> TimeInterval
 }
 
+/// Make your `AudioItem`-subclass conform to this protocol to control enable the ability to set initialization options for the asset.
```

Comment:
Make your `AudioItem`-subclass conform to this protocol ~~to control enable the ability~~ to set initialization options for the asset.

Reply:
Maybe also link to these [docs](https://developer.apple.com/documentation/avfoundation/avurlasset/initialization_options) to make it clear what options are to be provided.